### PR TITLE
Linguistic changes

### PR DESCRIPTION
**Pull request by ghost**
Hey @piecioshka !

Just added some revisions that I hope will make things a bit clearer for users. 

-R

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-## What it going on?
+## What's going on?
 
-I've write some unit test to describe some production code.<br/>
-Your mission is to **fill implementation** into application file.
+I've written some unit tests to describe sample production code.<br/>
+Your mission is to **complete the implementation** by completing a code file.
 
-## How to complete task?
+## How to complete thr task?
```
Comment (piecioshka) [inline README.md]: What does it mean?

**Comment by piecioshka** [comment]
@robreczarek thanks a lot!

I will merge when you fixed small typo ;-)

**Comment by piecioshka** [comment]
Thanks @robreczarek 🍺

**Comment by ghost** [comment]
@piecioshka  Awesome!!! 🎉 🎈

**Pull request merged**